integrations/context-ai.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: integration
+name: Context AI
+description: A component to log conversations for analytics by Context.ai - built for Haystack 2.0.
+authors:
+    - name: Alec Barber
+      socials:
+        github: BarberAlec
+        linkedin: https://www.linkedin.com/in/alec-barber/
+    - name: Alex Gamble
+      socials:
+        github: agamble
+        linkedin: https://www.linkedin.com/in/alex-gamble-13682086/
+    - name: Henry Scott-Green
+      socials:
+        linkedin: https://www.linkedin.com/in/hcscottgreen/
+    - name: Amishapriya Singh
+      socials:
+        github: amisha29sh
+        linkedin: https://www.linkedin.com/in/amisha29/
+pypi: https://pypi.org/project/context-haystack/
+repo: https://github.com/contextco/context-haystack
+type: Analytics Framework
```

Comment:
Heya, we have 2 tags that might work for you, can you please select the one that makes most sense: 
"Monitoring Tool" or "Evaluation Framework"

Reply:
Hi Tuana! Thanks, I have updated the type field to "Monitoring Tool".